changelogs/head.asciidoc

```diff
@@ -18,6 +18,7 @@ https://github.com/elastic/apm-server/compare/8.5\...main[View commits]
 [float]
 ==== Bug fixes
 - Null APM agent configuration entries are now treated as empty, and will not cause the server to exit with an error {pull}9546[9546]
+- Change ownership of apm-server.yml and data/ to apm-server user in Docker image {pull} 9604[9604]
```

Comment:
In case the fix gets shipped with `8.5.3` we'll need to remove from here again (but we can do that once we know).